### PR TITLE
Issue #2838 Update cmd-docker-env.feature file

### LINKED ISSUE #2838: Update cmd-docker-env.feature file
### General information

  * Minishift version: master
  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Expected
Remove the [step](https://github.com/minishift/minishift/blob/master/test/integration/features/cmd-docker-env.feature#L31). Maintenance effort of the feature file will be reduced

### Actual
Step ```Then stdout of host shell should not contain "Operating System: Minishift Boot2Docker ISO Version”``` making no sense to the scenario. This is a dynamic string phrase and we are putting it in a **SHOULD NOT CONTAIN** check which might keep changing even on next release iso version. Then it is obvious that the step passes for both upstream (Default centos iso) and downstream (rhel iso).

### Logs

You can start Minishift with `minishift start --show-libmachine-logs -v5` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

### PR DESCRIPTION
**Pull request by amitkrout**
Fixes #2838

Steps to test the pull request

  1. make integration GODOG_OPTS="-tags cmd-docker-env"
  2. 
  3. 
  4.

### CODE REVIEW

File: test/integration/features/cmd-docker-env.feature [hunk 1]
```diff
@@ -26,9 +26,8 @@ one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_SPECIFIED_SHELL parame
 
   Scenario: Unsetting Docker client using --unset flag of docker-env command
      When executing "minishift docker-env --unset" in host shell succeeds
-      And evaluating stdout of the previous command in host shell
+     Then evaluating stdout of the previous command in host shell
```
Comment (agajdosi) [inline test/integration/features/cmd-docker-env.feature]: This really should be `and`. I run unset the docker-env and I execute the command it prints to the console and I execute `docker info`. Then only I can expect something and that is that the output of `docker info` does not contain `Name: minishift`.

Comment (amitkrout) [reply test/integration/features/cmd-docker-env.feature]:
```
When
  And
 Then
  And
```
@agajdosi Is this the ordering you are looking for ? I feel this is better that the previous order

Comment (budhram) [reply test/integration/features/cmd-docker-env.feature]: @amitkrout , yes it is 
```
When
  And
  And (if similar test case)
Then
```

Comment (agajdosi) [reply test/integration/features/cmd-docker-env.feature]: In this case I think it should be as @budhram mentioned:
```
When
  And
  And (if similar test case)
Then
```
`And executing "docker info" in host shell` is not something we check, it is just part of WHEN (prolonged with AND), it is preparation, after witch THEN comes in which we actually really care what is the result, this is the actual check, why we do all of this.

**Comment by amitkrout** [comment]
Retest this please

File: test/integration/features/cmd-docker-env.feature [hunk 2]
```diff
@@ -27,8 +27,7 @@ one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_SPECIFIED_SHELL parame
   Scenario: Unsetting Docker client using --unset flag of docker-env command
      When executing "minishift docker-env --unset" in host shell succeeds
       And evaluating stdout of the previous command in host shell
-      And executing "docker info" in host shell
-     Then stdout of host shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
```
Comment (praveenkumar) [inline test/integration/features/cmd-docker-env.feature]: :+1: we have removed B2D quite some time now, good to remove this also.

**Comment by anjannath** [comment]
retest this please

**Comment by LalatenduMohanty** [comment]
retest this please

**Comment by LalatenduMohanty** [comment]
CentOS CI failed because of webconsole issue

https://ci.centos.org/job/minishift-pr/5110/console
```
on openshiftwebconsoleconfigs.webconsole.operator.openshift.io "instance": the object has been modified; please apply your changes to the latest version and try again
05:31:47 I1008 00:18:09.062733    2032 interface.go:41] Finished installing "openshift-web-console-operator" "openshift-image-registry" "openshift-router" "persistent-volumes" "centos-imagestreams" "sample-templates"
05:31:47 Error: Operation cannot be fulfilled on openshiftwebconsoleconfigs.webconsole.operator.openshift.io "instance": the object has been modified; please apply your changes to the latest version and try again
05:31:47 
```

**Pull request merged**